quickstart/java/jre/text-to-speech/src/speechsdk/quickstart/Main.java

```diff
@@ -24,46 +24,51 @@ public static void main(String[] args) {
         // Replace below with your own service region (e.g., "westus").
         String serviceRegion = "YourServiceRegion";
 
-        // Creates an instance of a speech synthesizer using speech configuration with specified
+        // Creates an instance of a speech synthesizer using speech configuration with
+        // specified
         // subscription key and service region and default speaker as audio output.
-        try (SpeechConfig config = SpeechConfig.fromSubscription(speechSubscriptionKey, serviceRegion);
-             SpeechSynthesizer synth = new SpeechSynthesizer(config)) {
-            
-            assert(config != null);
-            assert(synth != null);
+        try (SpeechConfig config = SpeechConfig.fromSubscription(speechSubscriptionKey, serviceRegion)) {
```

Comment:
Is it possible to just add `config.setSpeechSynthesisVoiceName("en-US-AriaNeural");` between that two lines?

Reply:
No, it will compile failed